docs/DeveloperGuide.md

```diff
@@ -316,6 +318,28 @@ The following sequence diagram shows how the reminder feature works:
 The following activity diagram summarises what happens when the user opens the application to see upcoming reminders:
 ![Activity diagram of the Reminder feature](images/ReminderActivityDiagram.png)
 
+### Set reminder window feature <a id="set-reminder-window-feature"></a>
+
+The `setWindow` feature allows users to set the reminder window to a specific number of days. Any roles stored in the company list that fall within the set window will be displayed in the reminder pane that automatically opens up on start.
+
+#### Implementation <a id="implementation-set-reminder-window"></a>
+
+![UML diagram of the SetWindow feature](images/SetReminderWindowDiagram.png)
+
+Given below is an example usage scenario and how the set reminder window feature behaves at each step:
+1. The user executes the command `setWindow 14` to set the reminder window to 14 days.
+2. Then the `SetReminderWindowCommandParser#parse()` creates an instance of `SetReminderWindowCommand` by passing the number of days to which the reminder window will be set.
+4. The `SetReminderWindowCommand#execute()` method will update the `model` using the `Model#setReminderWindow()` method, which consequently updates `model.userPrefs` by calling the `UserPrefs#setReminderWindow()` method to update the reminder window as stored in the user preferences.
+6. The `Parser` returns the `CommandResult` which is then executed by `LogicManager`.
```

Comment:
MInor fix, should be 5.

Reply:
Nice catch, I'll fix this right away 👍 thanks!